.github/workflows/_web-resources.yml

```diff
@@ -0,0 +1,42 @@
+name: "[Docker] Build web resources"
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  workflow_call:
+
+jobs:
+  build-web:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@f095bcc56b7c2baf48f3ac70d6d6782f4f553222 # pin@v3
+      - uses: actions/setup-node@869f4dd0c7f320ae834c2724d92a364de3893c24 # pin@v3.6.0
+        with:
+          node-version-file: "client/docker/resources/.nvmrc"
+      - name: cache node modules
+        id: cache-npm
+        uses: actions/cache@04f198bf0b2a39f7230a4304bf07747a0bddf146 # pin@v3.0.8
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ./web-resources/public
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/web-resources/package-lock.json') }}
+      - name: build assets
+        run: |
+          # Copy in templates and translations
+          cp -r ../app/templates ./templates
```

Comment:
Noticed this in passing. Templates are PHP/Twig code so shouldn't be needed in the node assets or the web server.

Reply:
yeah this was in the build image from before. I thought it seemed strange and tried removing it but it was still expecting a file somewhere... I must have been misunderstanding the message though as I have now removed that cp in and it still seems fine as you would expect :)